### PR TITLE
Create Research card issuing by PST.md

### LINKED ISSUE #19: Research card issuing by PST
@pavlovcik please review

### Intro 
PST https://pst.net/ is one of the top providers of global cards focused on advertisement.
They issue global Visa virtual cards for advertising and personal purchases. 
The KYC process is lightweight. 

### Goal
Research card issuing by PST 

### Activity

- **PST: gather general information** 

Get information on the PST cards specifics and pricing, collect public information about their card issuing process (realize how they managed to issue global cards), find their public partners (BIN sponsors, etc), other valuable information.

Time estimate: minimum 1 day.
Risks - not all required information can be available publicly. 

- **PST: research white-label solution and other options for cooperation** 

Contact PST, clarify the ability for us to use their white label solution, if it is possible then get terms and access. 

Time estimate: minimum 1 day.
Risks - they will not reply or not approve us because our request might not be relevant to them.

### PR DESCRIPTION
**Pull request by sergfeldman**
Resolves #19

### CODE REVIEW

File: Research card issuing by PST.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# Research card issuing by PST
+
+### PST cards specifics and pricing
+
+**Cards overview**
+
+PST issues virtual cards for advertising accounts, affiliate marketing, and personal purchases.
+They issue a few types of cards in USD & EUR.
+
+Per information from their support, according to the terms, cards can't be used to receive regular payments from third parties.
```
Comment (pavlovcik) [inline Research card issuing by PST.md:10]: Can you elaborate

File: Research card issuing by PST.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+At the same time, top-up from myself and in some cryptocurrencies is welcomed.
```
Comment (pavlovcik) [inline Research card issuing by PST.md:11]:
```suggestion
At the same time, top-up from the user and in some cryptocurrencies is welcomed. 
```

I don't understand

File: Research card issuing by PST.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+>Cards in USD and EUR accepted by stores and services worldwide
+
+https://pst.net/worldwide 
+
+**Cards pricing**
+
+>Ultima
+The most advanced virtual card in $ for all kinds of payments. With its high status and support of 3D-Secure technology, this card will become your trusted shopping assistant.
+0 $ transaction fee
+8% deposit fee
```
Comment (pavlovcik) [inline Research card issuing by PST.md:35]: This is insane. That's not usable 👀

File: Research card issuing by PST.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+6211 Security Brokers
+7273 Dating, escort services
+7995 Casino, Betting
+4829 Wires, Money Orders
+7800 Lotteries
+7801 Licensed Online Casino
+
+>Trials, malicious intent of attempting to charge virtual cards for amounts significantly exceeding (more than 40%) the available balance, transactions for a merchant account in order to reduce chargeback ratios, and microtransactions are strictly forbidden.
+
+>In case of violation of any mentioned restrictions, the system will deduct 30% of the transaction amount as a penalty from the account or card balance. It is impossible to cancel such a transaction. Also, in case of payment for service based in a restricted country, the system will block the card, with a 30% penalty of the transaction amount. The user needs to transfer the remaining funds back to the account balance during the 30-day period. Otherwise, the funds will be deducted to cover the bank expenses.
```
Comment (pavlovcik) [inline Research card issuing by PST.md:74]: These guys are crazy greedy

**Review by pavlovcik — COMMENTED** [review]
Yikes on their fees

File: Research card issuing by PST.md (hunk 1 above)
Comment (sergfeldman) [reply Research card issuing by PST.md:10]: The main point: these cards are not suitable for the fiat payouts to freelancers. 

Details:

This information is got from their support which is not talkative so information can be incomplete.

As I understand, the cards are aimed on spending, not collecting fiat funds.

It is welcomed to top up account in crypto and spend from a card in fiat.
Also, in their Private service, admins can top up team members' cards.

Potentially, a card owner can top up in fiat a card from another place where he passed KYC so the sender name and the receiver name is the same.
It is hard or not possible to use these cards for fiat payouts from third-parties where sender and receiver names are different.

Comment (pavlovcik) [reply Research card issuing by PST.md:10]:
```suggestion
Per information from their support, according to the terms, cards can't be used to receive regular payments from third parties.

This information is got from their support which is not talkative so information can be incomplete.

As I understand, the cards are aimed on spending, not collecting fiat funds.

It is welcomed to top up account in crypto and spend from a card in fiat.
Also, in their Private service, admins can top up team members' cards.

Potentially, a card owner can top up in fiat a card from another place where he passed KYC so the sender name and the receiver name is the same.
It is hard or not possible to use these cards for fiat payouts from third-parties where sender and receiver names are different.

```

**Review by pavlovcik — APPROVED** [review]

**Pull request merged**